src/apel_plugin.py

```diff
@@ -367,6 +376,15 @@ def create_sync_db(config, records):
                 sys.exit(1)
         else:
             site_name = site_id
+        try:
+            submit_host = r.meta.get(meta_key_submithost)[0].replace(
+                "%2F", "/"
+            )
+        except TypeError:
+            logging.warning(
+                f"No submithost found in record {r.record_id}, using default"
+            )
```

Comment:
Does it make sense to put this functionality into a dedicated function? You're using the exact code somewhere else as well. Putting it into a function avoids duplication and removes a potential future source of bugs.